src/app/ApplicationTemplate.Tests/Presentation/NavigationTestsBase.cs

```diff
@@ -14,6 +14,9 @@
 
 namespace ApplicationTemplate.Tests
 {
+	/// <summary>
+	/// Gives access to navigation events to reproduce the standard app behavior.
```

Comment:
The comment states giving access to navigation events but no navigation event are exposed.
It seems like this class offers helper methods for navigation-related tests.